src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+
+  border: 1vmin #000;

```

Comment:
The border property should specify both width and style together, e.g., `border: 1vmin dotted #000;` to avoid potential issues.